x-pack/plugins/security_solution/public/management/hooks/artifacts/use_bulk_delete_artifact.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import pMap from 'p-map';
+import { HttpFetchError } from 'kibana/public';
+import { ExceptionListItemSchema } from '@kbn/securitysolution-io-ts-list-types';
+import { useMutation, UseMutationResult, UseQueryOptions } from 'react-query';
+import { ExceptionsListApiClient } from '../../services/exceptions_list/exceptions_list_api_client';
+
+export function useBulkDeleteArtifact(
+  exceptionListApiClient: ExceptionsListApiClient,
+  customOptions: UseQueryOptions<ExceptionListItemSchema[], HttpFetchError>,
+  options: {
+    concurrency: number;
+  } = {
+    concurrency: 5,
```

Comment:
Do you remember if we have a common `const` for this? I thought we did, but maybe not

Reply:
No, I don't think so.